### PR TITLE
Updated guidelines for nomore than 7 col in a row

### LINKED ISSUE #25: 🟢 Update the Contribution Guidelines to reflect the new CONTRIBUTORS.md
Need to add a to the [CONTRIBUTION-GUIDELINES](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md) to mention that the [CONTRIBUTORS.md](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTORS.md) needs to explain how there is only to be a maximum of seven contributors per each column and that when a contributors is contributing and sees already column displayed above that they need to then close the </td> and create a new <td> tag. Give an example of this with a code block. 

🚨This explanation needs to be added to [Step 8 of the CONTRIBUTION-GUIDELINES](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md#step-8-add-to-the-contributorsmd-file)🚨

### Consider adding this text:

**To add a new row to the table in the given markdown file, follow these steps:**

1. Copy the last row of the table, including the <tr> and </tr> tags.
2. Paste the copied row below the last row of the table.
3. Replace the content of the cells in the new row with the information of the new contributor.
4. Make sure to update the column number in the <td> tags to match the number of columns in the table.

For example, if the last contributor was in column 7 and you want to add a new row with a new contributor, you would copy the last row of the table, paste it below the last row, and update the content of the cells with the information of the new contributor. You would also need to update the column number in the <td> tags to 8, since you are adding a new column.

- [ ] take away the alt text from the code in CONTRIBUTORS.md
- [ ] add to the CONTRIBUTION-GUIDELINES

### PR DESCRIPTION
**Pull request by Nawarajkarki**
## Issue Addressed
Closes #25

### CODE REVIEW

**Comment by XanderRubio** [comment]
Hi @Nawarajkarki 

Could you please provide more details about your pull request? I'm not entirely sure about the enhancements you're adding to the project. Please go ahead with first adding by sharing your before I die statement and then opening up a new pull request. Thank you!

**Comment by Nawarajkarki** [comment]
Hi @XanderRubio 

Thank you for your response.
The changes in my pull request are made addressing one of the open issues, (Issue number #25 ) OR  just click [here](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues/25) to view the issue

I have modified the [CONTRIBUTION-GUIDELINES](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md) explaining the Contributors to add maximum of 7 contributors per row.

According to the Open Issue the explanation was to be added to [Step 8 of the CONTRIBUTION-GUIDELINES](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md#step-8-add-to-the-contributorsmd-file)

## Go through these pages for a quick comparison 

### [Updated version](https://github.com/Nawarajkarki/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md#step-8-add-to-the-contributorsmd-file)


### [Current version](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md#step-8-add-to-the-contributorsmd-file)

**Comment by XanderRubio** [comment]
> Hi @XanderRubio
> 
> Thank you for your response. The changes in my pull request are made addressing one of the open issues, (Issue number #25 ) OR just click [here](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues/25) to view the issue
> 
> I have modified the [CONTRIBUTION-GUIDELINES](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md) explaining the Contributors to add maximum of 7 contributors per row.
> 
> According to the Open Issue the explanation was to be added to [Step 8 of the CONTRIBUTION-GUIDELINES](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md#step-8-add-to-the-contributorsmd-file)
> 
> ## Go through these pages for a quick comparison
> ### [Updated version](https://github.com/Nawarajkarki/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md#step-8-add-to-the-contributorsmd-file)
> ### [Current version](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md#step-8-add-to-the-contributorsmd-file)

Thank you for the clarification, @Nawarajkarki. I understand now. Does it seem clear to you how step 8 is set up to explain what to do if there are seven columns and a new contributor needs to start a new one? I ask because I would like to get feedback from another contributor to ensure that it is understandable. Thank you for your time. I agree with the changes you have made so far and want to confirm that step 8 is clear and organized in its steps upon first reading. Thank you for your help and for taking ownership of this issue. Well done! I await your feedback.

File: what.md
Comment (XanderRubio) [inline what.md:1]: Additionally, what is this file for?

Comment (Nawarajkarki) [reply what.md:1]: Oh! sorry for that . > 
That's nothing and I just deleted it.

**Review by XanderRubio — APPROVED** [review]
@Nawarajkarki Very good. Thank you for your assistance, and I added a screenshot for support. I am going ahead with merging now!

**Comment by XanderRubio** [comment]
Thank you @Nawarajkarki, for your time in assisting and contributing, and this closes out issue #25. Please go ahead and share what you want to do before you die! Have a nice day!

**Pull request merged**